### ISSUE TITLE
nuxt-strapi v1.1.0 support for nuxt 2

### ISSUE THREAD

**Issue by echatzief** [nuxt-modules/strapi#226]
Hey,

I want to ask if you will support nuxt 2 for the nuxt-strapi v1.1.0 in the future.

Thanks

**Comment by salv0** [comment]
Sorry to hijack your question, but mine is also related:
does the Nuxt 2 version support Strapi v4? or only v3?

It is not very clear from the docs

**Comment by luke-z** [comment]
@echatzief Nuxt2, as in v2.15.8 and lower is only compatible with the strapi module v0.
If you want to use v1, you need to install Nuxt3 or [Nuxt Bridge](https://v3.nuxtjs.org/getting-started/bridge/).

@benjamincanac improved compatibility with Nuxt Bridge in commit 70f73af. This is not yet released in v1.2.0, so for the moment you'll need to use the [edge channel](https://strapi.nuxtjs.org/setup#edge-channel).

--------------------------------

@salv0 v0 of this module only supports Strapi3

**Comment by MexsonFernandes** [comment]
I have released a module to support Nuxt 2 and Strapi 4: https://www.npmjs.com/package/nuxt2-strapi4.

In my team I had project which depends on nuxt 2 completely and migration was breaking old system. The Nuxt Strapi module older one is much better for us as we have token refresh for our mobile app. So I forked https://github.com/nuxt-modules/strapi module to support for Strapi 4. 

Feel free to give me some suggestions and feedback to improve it. I will maintain this as its required in my other projects as well.
